string semantic_class
float64 object_score
geometry_msgs/Point[] bbox_3d